lib/cf/broker/src/auth/oauth.js

```diff
@@ -18,12 +18,12 @@ const init = () => {
 
   debug('Fetching OAuth system token from server %o', config.uris().api);
   systemToken.start((err) => {
-    if (err)
+    if (err) {
       edebug('Could not fetch OAuth token %o', err);
-    else
-      debug('Successfully fetched OAuth token from %s', config.uris().api);
+      throw new Error('Could not fetch OAuth token');
```

Comment:
This will restart the broker. We end up with endless loop if the app is not configured